docker-compose.production.yaml

```diff
@@ -13,7 +13,7 @@ services:
       - postgres
       - redis
     ports:
-      - 3000:3000
+      - '127.0.0.1:3000:3000'
```

Comment:
@vishnu-narayanan  should we enable the binding for 3000 alone to be open. It should be relatively harmless considering it's just the webserver.  
+ it will make it easy for someone to verify the installation on a VPS before setting up a frontend proxy like Nginx?

Reply:
@sojan-official Since this is the production file, I prefer we go with the secure default. To verify the installation, one can do something like, `curl -I localhost:3000` or change the production compose file to drop the `127.0.0.1` binding.

Reply:
sounds good 👍